src/schema/objects/files.yaml

```diff
@@ -126,3 +126,8 @@ stimuli:
     A directory to store any stimulus files used during an experiment.
     See the [relevant section](SPEC_ROOT/modality-specific-files/task-events.md#stimuli-directory)
     for more information.
+descriptions:
+  display_name: Descriptions Definitions
+  file_type: regular
+  description: |
+    A TSV file describing labels found for the `desc` entity in a Derivatives dataset.
```

Comment:
This actually might conflict with the suffix definition later. Should I remove it?

Reply:
Removing it.